src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import cn from 'classnames';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  hasActiveTodo: boolean;
+  filterTodos: (filterBy: FilterStatus) => void;
+  filterStatus: FilterStatus,
+  countOfLeftTodos: number,
+};
+
+export const TodoFooter:React.FC<Props> = React.memo(({
+  hasActiveTodo,
+  filterTodos,
+  filterStatus,
+  countOfLeftTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${countOfLeftTodos} items left`}
+      </span>
```

Comment:
can we define the value of todos inside this component instead of passing it via props?